src/libraries/System.Private.CoreLib/src/System/Collections/Generic/HashSet.cs

```diff
@@ -1068,6 +1056,69 @@ private int Initialize(int capacity)
             return size;
         }
 
+        /// <summary>
+        /// Checks if trim excess is needed based on ratio of entries length to count
+        /// </summary>
+        /// <returns></returns>
+        private bool IsTrimNeeded()
+        {
+            return _count > 0 && _entries!.Length / _count > ShrinkThreshold;
+        }
+
+        /// <summary>
+        /// Initializes the HashSet from another HashSet with the same element type and equality comparer
+        /// </summary>
+        /// <param name="source"></param>
+        private void ConstructFrom(HashSet<T> source)
```

Comment:
Why has this method been moved from this original location? It's more difficult to review any changes that might have been made to it

Reply:
Construct From moved from it's original location as it's not an initialization method anymore. Also CopyEntries separated from TrimExcess as it's needed by ConstructFrom

Reply:
It doesn't technically need to be moved though. Keeping it at its originally location has the benefit of being easier to review and better preserving edit history.

Reply:
Got it, it won't happen again

Reply:
Can you please move it back / revert it?